ui/cryptomaterial/dropdown.go

```diff
@@ -134,69 +152,78 @@ func (d *DropDown) Changed() bool {
 	return false
 }
 
-func (d *DropDown) layoutActiveIcon(gtx layout.Context, index int) D {
+func (d *DropDown) layoutActiveIcon(gtx C, index int) D {
 	var icon *Icon
 	if !d.isOpen {
 		icon = NewIcon(d.dropdownIcon)
 	} else if index == d.selectedIndex {
 		icon = NewIcon(d.navigationIcon)
 	}
 
-	return layout.E.Layout(gtx, func(gtx C) D {
-		if icon != nil {
-			icon.Color = d.theme.Color.Gray1
-			return icon.Layout(gtx, values.MarginPadding20)
-		}
-		return layout.Dimensions{}
-	})
+	if icon == nil {
+		return D{} // return early
+	}
+
+	if !d.isOpen || d.SelectedItemIconColor == nil {
+		icon.Color = d.theme.Color.Gray1
+	} else {
+		icon.Color = *d.SelectedItemIconColor
+	}
+
+	return icon.Layout(gtx, values.MarginPadding20)
 }
 
-func (d *DropDown) layoutOption(gtx layout.Context, itemIndex int) D {
+func (d *DropDown) layoutOption(gtx C, itemIndex int) D {
 	item := d.items[itemIndex]
 	radius := Radius(0)
 	clickable := item.clickable
 	if !d.isOpen {
 		radius = Radius(8)
 		clickable = d.clickable
+		clickable.Hoverable = d.MakeSelectedItemHoverable
```

Comment:
This isn't being applied to the selected item alone, sure `MakeSelectedItemHoverable` is the right name to use?

Also, should this be set at the point where item.clickable was initialized, rather than on each layout?

Reply:
> This isn't being applied to the selected item alone,

It is, but only when the dropdown is closed.

> sure MakeSelectedItemHoverable is the right name to use

Any suggestion? I'm sure the name fits its use.

> Also, should this be set at the point where item.clickable was initialized, rather than on each layout?

This special behaviour is only required when the dropdown is closed but if it's open, all items are hovarable.